font-test-data/src/lib.rs

```diff
@@ -375,3 +375,7 @@ pub mod ift {
         0x00, 0x00,               // last_entry_index
     ];
 }
+
+pub mod ttc {
+    pub static TTC: &[u8] = include_bytes!("../test_data/ttc/TTC.ttc");
```

Comment:
Similar to practices in Chromium or our other open source projects, I would suggest to create a `test_data/third_party/` or `test_data/third_party/harfbuzz` directory, and please a readme with provenance and license there, so we have a clear indication where the font is coming from.

Reply:
I agree with that but as this applies to all our testdata I have defered it to https://github.com/googlefonts/fontations/issues/1127